lib/xcodeproj/project/object/build_phase.rb

```diff
@@ -29,6 +29,16 @@ class AbstractBuildPhase < AbstractObject
         #         installing`.
         #
         attribute :run_only_for_deployment_postprocessing, String, '0'
+        
+        # @return [String] whether or not this run script will be forced to 
+        #         run even on incremental builds. Can be either '1', or 
+        #         missing. By default this option is disabled in Xcode.
+        #
+        # @note   This setting is exposed in Xcode in the UI of
+        #         PBXShellScriptBuildPhase as `Based on
+        #         dependency analysis` (selected by default).
+        #
+        attribute :always_out_of_date, String
```

Comment:
shoudl the default value be '0' like in the one above? Or is that already the case?

Reply:
the default is for the whole option doesn't appear in the pbxproj. Only when you disable it it comes up as having the value 1.

Reply:
sounds good